core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.ts

```diff
@@ -118,6 +119,8 @@ export class InputResponsePairComponent {
       (this.audioPlayerService.isPlaying() ||
       this.autogeneratedAudioPlayerService.isPlaying())) {
       return ExplorationPlayerConstants.AUDIO_HIGHLIGHT_CSS_CLASS;
+    } else {
+      return '';
```

Comment:
Why is this added?

Reply:
Without adding, this, we get the error that "function lacks ending return statement and return type does not include 'undefined'." . Just wanted to ask, is there a better way to handle this?

Reply:
Okay, I think this is a good way to handle this.